website/docs/docs/develop/develop-in-the-cloud.md

```diff
@@ -0,0 +1,249 @@
+
+### Prerequisites
+
+To develop in dbt Cloud IDE (beta), you need to meet these requirements:
+
+- Your dbt project must be compatible with dbt v0.15.0. The dbt IDE is powered by the [dbt-rpc](/reference/commands/rpc) which was overhauled in dbt v0.15.0
+- You must have a [Developer License](/docs/dbt-cloud/access-control/cloud-seats-and-users)
+- - Your dbt project must be connected to a [data platform](/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database)
+- You must have a [development environment and development credentials](/docs/develop/develop-in-the-cloud#set-up-the-cloud-ide) set up
+- The environment must be on dbt version 1.0 or higher
+- Currently only multi-tenant instances of dbt Cloud can develop in the updated beta version of the Cloud IDE
+    * Single-tenant instances will soon be able to opt into this Beta release.
+- Your dbt repository (in dbt Cloud) must have write access enabled
+    * See [Connecting your GitHub Account](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application) and [Importing a project by git URL](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url) for detailed setup instructions
+
+
+The dbt Cloud IDE beta brings the startup and interaction time for dbt project development down from minutes to seconds, includes reliable performance and delightful enhancements to help you develop more efficiently!
+
+### Start up and work retention
+
+<table>
+<tr><th>Start up process </th><th>Work retention</th></tr>
+<tr><td>
+
+There are three start-up states when using or launching the Cloud IDE:
+    
+- Creation start - This is the state in which you start the IDE for the first time. You can also view this as a *cold start* (see below), and you can expect this state to take longer because the git repository is being cloned.
+- Cold start - This is the process of starting a new develop session, which will be available for you for three hours. The environment automatically turns off three hours after the last activity with the rpc server. This includes compile, preview, or any dbt invocation, however, it *does not* include editing and saving a file.
+- Hot start -  This is the state of resuming an existing or active develop session within 3 hours of the last activity. 
+    
+
+</td><td>
+    
+
+The Cloud IDE needs explicit action to save your changes and there are three ways your work is stored:
+
+- Unsaved, local code - Any code you write is automatically available from your browser’s storage. You can see your changes but will lose them if you switch branches or browsers (another device or browser).
+- Saved but uncommitted code - When you save a file, the data gets stored in your local storage (EFS storage). If you switch branches but don’t *commit* your saved changes, you will lose your changes.
+- Committed code - This is stored in the branch with git provider and you are able to check out other (remote) branches. 
+
+
+</td></tr> </table>
```

Comment:
great side-by-side comparison table! 

is it possible to make them equal widths? it renders uneven for some reason:

![Screen Shot 2022-10-12 at 5 31 38 PM](https://user-images.githubusercontent.com/107218380/195472559-e34aff74-724b-4d6d-9baa-aaebd5e4f532.png)